Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php

```diff
@@ -91,7 +91,7 @@ private static function encloseTextInComment($header)
         foreach ($lines as $line) {
             $comment .= rtrim(' * '.$line)."\n";
         }
-        $comment .= " */\n";
+        $comment .= ' */';
```

Comment:
Ok, then. What I was getting at is that this is unrelated, and should already be fixed on 1.7. I assumed it was you see.